packages/theme-wizard-templates/src/utils/token-groups.ts

```diff
@@ -0,0 +1,60 @@
+type TokenValue = {
+  path?: string[];
+  [key: string]: unknown;
+};
+
+type TokenScale = Record<string, unknown>;
+
+const isTokenValue = (value: unknown): value is TokenValue =>
+  Boolean(value && typeof value === 'object' && 'path' in value && Array.isArray((value as TokenValue).path));
+
+const visitTokenTree = (node: unknown, onToken: (token: TokenValue) => void): void => {
+  if (!node || typeof node !== 'object') {
+    return;
+  }
```

Comment:
Deze checks worden ook al gedaan in `isTokenValue`, dus hier misschien overbodig?